src/librustc_target/abi/mod.rs

```diff
@@ -716,10 +716,10 @@ pub enum Variants {
     },
 
     /// General-case enums: for each case there is a struct, and they all have
-    /// all space reserved for the discriminant, and their first field starts
-    /// at a non-0 offset, after where the discriminant would go.
+    /// all space reserved for the tag, and their first field starts
+    /// at a non-0 offset, after where the tag would go.
     Tagged {
-        discr: Scalar,
+        tag: Scalar,
         variants: Vec<LayoutDetails>,
```

Comment:
I'm not sure if this declaration of `Tagged` here is describing the in memory representation of a tagged union, but my code wouldn't compile because of something I changed in `layout.rs` I think. So I made the change as shown here.

Reply:
That's fine, this enum seems to have been moved from librustc to here